packages/base-contract/src/index.ts

```diff
@@ -61,6 +68,8 @@ export class BaseContract {
     public address: string;
     public contractName: string;
     public constructorArgs: any[] = [];
+    private _evmIfExists?: any; // hack (xianny): VM is also used as a namespace
```

Comment:
Can you elaborate on the need to use `any` here?

Reply:
The type should be `VM`, but `ethereumjs-vm` defines it as a namespace and the ts compiler doesn't allow it to be used as a type. I tried importing it all different ways and couldn't figure it out :/

Reply:
Aha! there was a leftover module declaration in `base-contract`. removing it fixed it.